doc/nrf/protocols/bt_mesh/nlc.rst

```diff
@@ -0,0 +1,30 @@
+.. _ug_bt_mesh_nlc:
+
+Networked Lighting Control profiles
+###################################
+
+Bluetooth® Networked Lighting Control (NLC) profiles are a set of device profiles built on top of the Bluetooth mesh protocol, specified by the Bluetooth Special Interest Group (SIG) in the `Bluetooth NLC profile specifications`_.
```

Comment:
NLC Profiles are ... specified by SIG in NLC profiles? Sounds a bit weird.

```suggestion
Bluetooth® Networked Lighting Control (NLC) profiles are a set of device profiles built on top of the Bluetooth mesh protocol, specified by the Bluetooth Special Interest Group (SIG).
```

Reply:
Well, now there is no link to the spec download site anymore. If you have a proposal that doesn't sound weird but keeps the link, I'm open for changing it, but I'd like to keep the link.

Reply:
How about?

```suggestion
`Bluetooth NLC profile specifications`_, specified by the Bluetooth Special Interest Group (SIG), are a set of device profiles built on top of the Bluetooth mesh protocol.
```

Reply:
I'm not really a fan of opening with a link like that. I also don't see the problem with a sentence on the form "X is specified by Y in the X specification". Is "C is a programming language specified by ANSI in the `C language specification`" also weird to you? Because I think it's fine.

@mia-ko what do you think here?

Reply:
Then I will leave it upto Mia to decide.

Reply:
I think I am ok with the original formulation. I don't see why we should have an issue with NLC profiles being specified by SIG in an NLC spec.
Might it be 'specified by' that is making it not sound right Omkar? How about if we changed to defined by?
```suggestion
Bluetooth® Networked Lighting Control (NLC) profiles are a set of device profiles built on top of the Bluetooth mesh protocol, defined by the Bluetooth Special Interest Group (SIG) in the `Bluetooth NLC profile specifications`_.
```

Reply:
Well, profiles are always defined in specifications. There is no profile that is not defined in a specification document. So in my mind, when I read 'profiles', I read it as 'profile specifications'.

If you think it is ok, then I have no issues. This is not at all a blocker.